app/views/content/teacher-training-advisers/_article.html.erb

```diff
@@ -92,15 +92,6 @@
       <li>find a role</li>
     </ul>
 
-      <%= render Content::QuoteListComponent.new(
```

Comment:
Shouldn't we remove the section above about returning to teaching as well as the quote @philspray @EllieNodder ?